app/views/shared/_search_bar.html.erb

```diff
@@ -1,12 +1,17 @@
-<div class='c-input-group o-grid__item' data-controller='search' span@medium='2-6' span='3-5'>
-  <div class='c-input-group__icon'>
-    <%= icon_tag 'search', size: 'large' %>
-  </div>
-  <input id='js-search-input' class='c-input' data-search-target='input' data-action='keydown->search#query' data-test-id='search_input' value='<%= params[:query] %>' />
-  <div class='c-input-group__icon u-display-none' data-search-target='loader'>
-    <%= loader_tag size: 'small' %>
-  </div>
-</div>
+<%= form_tag search_path, method: :get, class: 'c-input-group o-grid__item', 'span@medium': '2-6',
+             span: '3-5', id: 'search',
+             data: { controller: 'search',
+                     action: 'turbo:submit-start->search#loading turbo:submit-end->search#loading search#submit' } do %>
+    <div class='c-input-group__icon'>
+      <%= icon_tag 'search', size: 'large' %>
+    </div>
+    <%= text_field_tag 'query', params[:query], class: 'c-input',
+                       autofocus: params[:query].present?, data: { search_target: 'input' } %>
```

Comment:
Input field autofocus attribute can not put the cursor to the end of the input field.